### ISSUE TITLE
Print out scope terser

### ISSUE THREAD

**Issue by dwijnand** [lightbend/dbuild#194]
I don't have an example handy (@sethtisue might do), but I've seen dbuild print out pages and pages of what is effectively sbt's Scope toString, which is falling back to the default case class toString and printing things like `Select(..)`.

It would be a massive enhancement if instead of toString-ing it ran that through `sbt.Def.displayFull` to get a nice, terse string output.

**Comment by dwijnand** [comment]
eg.

```
 ScopedKey(Scope(Select(ProjectRef(file:/home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.7-RC1/project-builds/scalafix-f453bc454adee97e9edb70cc85ce3e04d01f0f88/,scalafix-nsc)),Global,Global,Global),moduleSettings)
```

in https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1255/console

**Comment by dwijnand** [comment]
Took a swing at fixing this: https://github.com/lightbend/dbuild/pull/202

**Comment by dwijnand** [comment]
Fixed in https://github.com/lightbend/dbuild/pull/204.

**Comment by dwijnand** [comment]
shipped in v0.9.10 if any dbuild user out there wants to upgrade!

**Comment by SethTisue** [comment]
I upgraded the Scala community build, and ran it, and a project failed, and dbuild didn't barf up something the size of Cincinnati onto the carpet! thanks! 👍
